### DISCUSSION TITLE
希望集成国产xmake作为默认的构建系统

### DISCUSSION THREAD

**Discussion by XIE-Yao** [linuxdeepin/developer-center#5450] [Features Request & Ideas | 特性请求 & 头脑风暴]
cmake虽然用户广, 但是真不值得去深入研究, 语法逻辑烂的让人窒息. 这方面xmake就好太多, 用lua脚本作为配置语言, 有配置域和脚本域两种功能, 可集成性也超越cmake太多, 关键编译速度还快. 

嵌入式起家, 各种语言都支持, 我的工业软件几十万行也都用它构建.

强烈推荐:)

**Comment by BLumia** [comment]
你指的是 deepin-unioncode 集成还是 deepin 发行版集成？后者的话即便集成也不会有“默认”的概念呀？

**Comment by Decodetalkers** [comment]
xmake 支持pkgconfig和cmake

**Comment by xzl01** [comment]
不会作为默认构建系统，但是以后可能作为一个可选项进行提供。因为cmake已经成为了构建系统的事实标准了
